src/Core/Entities/IReferenceable.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace Bit.Core.Entities;
+﻿#nullable enable
+
+namespace Bit.Core.Entities;
 
 public interface IReferenceable
 {
     Guid Id { get; set; }
-    string ReferenceData { get; set; }
+    string? ReferenceData { get; set; }
```

Comment:
This doesn't map to a sql table, so harder to confirm. What do you think about removing this change?

Reply:
It doesn't map to a table directly but the only things that uses it *are* tables and it isn't required on those tables. Therefore the interface should have the lowest common denominator.